aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -448,6 +458,32 @@ protected boolean isRdsCustomOracleInstance(final ResourceModel model) {
         return RDS_CUSTOM_ORACLE_ENGINES.contains(model.getEngine());
     }
 
+    protected List<Event> fetchEvents(
+            final ProxyClient<RdsClient> rdsProxyClient,
+            final ResourceModel model,
+            final String eventType,
+            final Instant fetchSince
+    ) {
+        DescribeEventsResponse response = rdsProxyClient.injectCredentialsAndInvokeV2(
+                DescribeEventsRequest.builder()
```

Comment:
I think we are missing `duration` parameter from the describe call. It defaults to 60 minutes. If any event happens after 60 minutes since the start of the update handler, we will fail to detect it.

Ideally, we would also want logical clocks instead of UTC ones, but alas RDS API does not provide that functionality :(